moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -1502,6 +1505,12 @@ moveit::planning_interface::MoveItErrorCode moveit::planning_interface::MoveGrou
   return impl_->move(false);
 }
 
+actionlib::SimpleActionClient<moveit_msgs::MoveGroupAction>&
+moveit::planning_interface::MoveGroupInterface::getMoveGroupClient()
```

Comment:
This method is missing the const  specifier, while the implementation method has one.
```suggestion
moveit::planning_interface::MoveGroupInterface::getMoveGroupClient() const
```

Reply:
Good catch, I obviously meant to make it const everywhere. Fixed!